lib/membrane_rtc_engine/endpoints/webrtc/bandwidth_manager.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Membrane.RTC.Engine.Endpoint.WebRTC.BandwidthManager do
```

Comment:
TrackReceiver uses `connection_prober` and `connection_prober_module` terms everywhere so we should either rename connection_prober to bandwidth_manager or rename this behavior to ConnectionProber. 

While ConnectionProber in fact also splits bandwidth I think that its name is more specific and also it is commonly used in other sfu implementations so maybe let's stick with it?

Reply:
The idea was that ConnectionProber is a very specific BandwidthManager that well, manages bandwidth, but is also capable of producing more of it